packages/cli/src/services/project-updater.ts

```diff
@@ -0,0 +1,20 @@
+import * as fs from 'fs-extra'
+import * as path from 'path'
+
+export async function updatePackageJsonDependencyVersions(version: string, projectPath: string): Promise<void> {
+    const projectAbsolutePath = path.resolve(projectPath)
+    const packageJsonPath = path.join(projectAbsolutePath, 'package.json')
+    try {
+        const packageJsonContents = require(packageJsonPath)
+        packageJsonContents.dependencies['@boostercloud/framework-core'] = `^${version}`
+        if (packageJsonContents.dependencies['@boostercloud/framework-types'] !== undefined) {
+            packageJsonContents.dependencies['@boostercloud/framework-types'] = `^${version}`
```

Comment:
isn't it better to remove the `^` symbol? We're checking versions anyway, right? 🤔 

Also, will rockets get updated with this?

Reply:
I used ^ symbol because it appears in all project's package.json files. I think I can replace all instances of @boostercloud with a loop instead

Reply:
@juanjoman I implemented the method in a better way. Nonetheless, we are not using it, since it's better to make the user update package.json by themself